app/Observers/EndorsementObserver.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Observers;
+
+use App\Events\Mship\Endorsement\TierEndorsementAdded;
+use App\Models\Atc\PositionGroup;
+use App\Models\Mship\Account\Endorsement;
+
+class EndorsementObserver
+{
+    /**
+     * Handle the Endorsement "created" event.
+     */
+    public function created(Endorsement $endorsement): void
+    {
+        $endorsement->load('account');
+
+        if ($endorsement->endorsable_type == PositionGroup::class) {
```

Comment:
Might look nicer as a match statement to make future expansion easier

Reply:
Agreed - another change to come soon for solo endorsements where pattern matching is going to be super helpful.